src/main/java/org/craftercms/studio/impl/v2/service/policy/internal/PolicyServiceInternalImpl.java

```diff
@@ -55,7 +55,7 @@ public class PolicyServiceInternalImpl implements PolicyServiceInternal {
     public static final String CONFIG_KEY_PERMITTED = "permitted";
     public static final  String CONFIG_KEY_DENIED = "denied";
 
-    protected ContentRepository contentRepository;
+    protected GitContentRepository contentRepository;
```

Comment:
I thought we wanted the higher services to not know about Git Repo nor care what the repo persistence mechanism is. Why make this Git aware?

Reply:
This will be revisited on API v1->v2 updates